Amplify/Categories/DataStore/DataStoreCategory+Behavior.swift

```diff
@@ -20,8 +20,7 @@ extension DataStoreCategory: DataStoreBaseBehavior {
 
     public func query<M: Model>(_ modelType: M.Type,
                                 byIdentifier id: String,
-                                completion: (DataStoreResult<M?>) -> Void) where M: ModelIdentifiable,
-                                                                                 M.IdentifierFormat == ModelIdentifierFormat.Default {
+                                completion: (DataStoreResult<M?>) -> Void) where M: ModelIdentifiable, M.IdentifierFormat == ModelIdentifierFormat.Default {
```

Comment:
let's revert this formatting as well for consistency across the rest of the formatting in this file